src/ontology/uberon-edit.obo

```diff
@@ -224207,6 +224207,43 @@ relationship: dc-contributor https://orcid.org/0000-0002-0819-0473
 relationship: located_in UBERON:0002371 ! bone marrow
 property_value: dcterms-date "2023-06-26T13:48:30Z" xsd:dateTime
 
+[Term]
+id: UBERON:8480071
+name: skin of lobe of tail
+def: "A zone of skin that is part of the lobe tail." [https://orcid.org/0000-0002-0819-0473]
+synonym: "fluke skin" NARROW [PMID:21959845]
+is_a: UBERON:0000014 ! zone of skin
+relationship: dc-contributor https://orcid.org/0000-0002-0819-0473
+relationship: part_of UBERON:0013131 ! lobe of tail
+property_value: dcterms-date "2023-08-29T09:37:23Z" xsd:dateTime
+
+[Term]
+id: UBERON:8480072
+name: skin of shin
+def: "A zone of skin on the front hindlimb zeugopod." [https://orcid.org/0000-0002-0819-0473]
+is_a: UBERON:0000014 ! zone of skin
+relationship: dc-contributor https://orcid.org/0000-0002-0819-0473
+relationship: part_of UBERON:0003823 ! hindlimb zeugopod
+property_value: dcterms-date "2023-08-29T09:47:32Z" xsd:dateTime
+
+[Term]
+id: UBERON:8480073
+name: skin of scapula region
+def: "A zone of skin that is above the scapula in the upper back region." [https://orcid.org/0000-0002-0819-0473]
+is_a: UBERON:0000014 ! zone of skin
+relationship: dc-contributor https://orcid.org/0000-0002-0819-0473
+relationship: part_of UBERON:0008231 ! dorsal thoracic segment of trunk
```

Comment:
consider using [upper back skin](http://purl.obolibrary.org/obo/UBERON_0009015) (it is part_of dorsal thoracic segment of trunk)

Reply:
@aleixpuigb does this term require only is_a 'upper back skin' without any additional relationship: part_of ?

Reply:
I would say that the skin of the scapula region is part_of some upper back skin.

Reply:
@aleixpuigb OK. Should I also add here a logical definition 'zone of skin' and ('part of' some 'upper back skin')?

Reply:
OK

Reply:
> @aleixpuigb does this term require only is_a 'upper back skin' without any additional relationship: part_of ?

Sorry @paulacalipho, you were right in this comment. Looking at other examples, a scapular skin is_a 'upper back skin'.